py3status/modules/whatismyip.py

```diff
@@ -52,6 +53,7 @@
 
 URL_GEO_OLD_DEFAULT = 'http://ip-api.com/json'
 URL_GEO_NEW_DEFAULT = 'https://ifconfig.co/json'
+UA = 'Mozilla/5.0 py3status'
```

Comment:
I think we should add the hostname of the local machine to avoid having all py3status users use the same UA. If a user spams, it should not impact others

On a better version we could even generate a random uuid stored in the py3 storage and use it on the UA as well to add randomness

Reply:
Is it the core which should generate py3status uuid for all modules or each module shoud manage there?

Reply:
Thus, about using hostname even uuid, not really like this as a paranoid. About privacy api owner can easily track user move.
What do you think about using a timestamp in UA?

Reply:
A hostname is nothing like a PII, API owners can already track your IP address which is considered a PII, so if privacy is your concern it has nothing to do with a hostname

As for the UUID, the module should

Reply:
A thought... Use `cache_timeout` instead of `negative_cache_timeout`... And take away `self.py3.error` in favor of a warning note about not abusing the API as I don't think we have any API modules that prevents users from specifying what they want in `cache_timeout`. We just write warnings. We also avoid a new configuration too.

weather_owm.py:
```
    cache_timeout: The time between API polling in seconds
        It is recommended to keep this at a higher value to avoid rate
        limiting with the API's.
        (default 600)
```
bitcoin_price.py
```
    cache_timeout: refresh interval for this module. A message from
        the site: Don't query more often than once every 15 minutes
        (default 900)
```
coin_markets.py
```
    cache_timeout: refresh interval for this module. a message from the site:
        please limit requests to no more than 10 per minute. (default 600)
```

Unrelated: I wonder if running `dropbox`, `yandexdisk_status` commands
even query the API every 10 seconds.

Reply:
Agreed, we should get rid of this option